backend/model.js

```diff
@@ -9,4 +9,4 @@ const noun = new Schema({
     total_count: Number
 });
 
-module.exports = mongoose.model("noun", noun, "noun");
\ No newline at end of file
+module.exports = mongoose.model("noun", noun, "noun");
```

Comment:
what is this

Reply:
The call to `mongoose.model` ties to the same name of the collection in MongoDB. To use `noun` document in MongoDB, the call should look like the above.